src/styles/blocks/card.scss

```diff
@@ -0,0 +1,134 @@
+.card {
+    font-family: Roboto, sans-serif;
+    font-weight: 500;
+    width: 200px;
+    height: fit-content;
+    box-sizing: border-box;
+    margin: 0;
+    padding: 32px 16px 16px;
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border-radius: 5px;
+    border: 1px solid #F3F3F3;
+    transition: transform $animation-duration ease;
+
+    &:hover {
+      transform: scale(1.20);
+    }
+  
+    &__img  {
+      width: 160px;
+      height: 134px;
+    }
+  
+    &__info-area {
+      margin: 40px 0 0;
+      padding: 0;
+      display: flex;
+      flex-direction: column;
+      color: #060b35;
+    }
+  
+    &__title {
+      font-size: 12px;
+      font-weight: 500;
```

Comment:
The card title should have a bigger font-weight:
![image](https://github.com/mate-academy/layout_catalog/assets/67425140/b7d47c84-e638-432f-bd04-4e22ed690a01)

Reply:
Hi @jv-aquino, this value of 500 is a requirement in [figma mockup](https://www.figma.com/file/ojkArVazq7vsX0nbpn9CxZ/Moyo-%2F-Catalog-(ENG)?type=design&node-id=32249-354&mode=design&t=dSP2bSaNYNtutBd1-0). Let me know if is really necessary change this value. If yes, can you provide the correct value?